File: Sesion 6/Actividades/Actividad 2/Actividad 2 S6.java
class Circulo {
    private double radio;
    
    public Circulo(double r) {
        this.radio = r;
    }
    
    public double obtenerRadio() {
        return this.radio;
    }
}

public class Main {
    public static double calcularArea(Circulo c) {
        final double pi = 3.1416;
        return pi * c.obtenerRadio() * c.obtenerRadio();
    }

    public static void main(String[] args) {
        Circulo circulo1 = new Circulo(5.0);
        Circulo circulo2 = new Circulo(7.0);

        System.out.println("Área del círculo 1: " + calcularArea(circulo1));
        System.out.println("Área del círculo 2: " + calcularArea(circulo2));
    }
}

File: Sesion 6/Actividades/Actividad 2/Actividad 2 S6.cpp
#include <iostream>
using namespace std;

class Circulo {
private:
    double radio;
public:
    Circulo(double r) : radio(r) {}
    double obtenerRadio() {
        return radio;
    }
    friend double calcularArea(Circulo &c);
};


double calcularArea(Circulo &c) {
    const double pi = 3.1416;
    return pi * c.radio * c.radio;
}


int main() {
    Circulo circulo1(5.0); 
    Circulo circulo2(7.0); 
    
    cout << "Área del círculo 1: " << calcularArea(circulo1) << endl;
    cout << "Área del círculo 2: " << calcularArea(circulo2) << endl;
    
    return 0;
}
